src/utils/sources-tree/getURL.js

```diff
@@ -54,13 +55,14 @@ function _getURL(source: Source, defaultDomain: string): ParsedURL {
         group: `${protocol}//${host || ""}`
       };
 
-    case "webpack:":
     case "ng:":
+    case "webpack:":
       return {
         ...def,
         path: pathname,
         filename,
-        group: `${protocol}//`
+        group: `${protocol}//`,
+        isAbsolutePath: pathname.startsWith("//")
```

Comment:
This doesn't seem right to me, if we're using WHATWG URL to process this, it should just be a single slash.

Reply:
@loganfsmyth  For webpack, we can have urls with double slash, and dots
Please find some webpack urls below:

 webpack:///webpack/bootstrap fc4578a0bef29d2a63f3 
 webpack:///src/component/index.js 
 webpack:////Users/workspace/github/debugger.html/node_modules/react/index.js 
 webpack:///./utils/debug.js 
 webpack:///../node_modules/devtools-environment/test-flag.js

Reply:
In these examples, with `whatwg-url`, the `pathname` should be

* `/webpack/bootstrap fc4578a0bef29d2a63f3`
* `/src/component/index.js`
* `//Users/workspace/github/debugger.html/node_modules/react/index.js`
* `/utils/debug.js`
* `/node_modules/devtools-environment/test-flag.js`

what is throwing me off here is these are already all absolute because they start with a single slash.

Reply:
I think part of what is tripping me up is, aren't those examples just mistakes on the part of Webpack? All URLs like this are absolute. The fact that there is a double slash at the start of one of them just means that one of the path segments is an empty string. It seems like a mistake in whatever configuration generated that URL. It seems unreasonable to expect the devtools to try to interpret what is essentially bad data.

With URLs as
```
webpack:///webpack/bootstrap fc4578a0bef29d2a63f3
webpack:///src/component/index.js
webpack:////Users/workspace/github/debugger.html/node_modules/react/index.js
webpack:///./utils/debug.js
webpack:///../node_modules/devtools-environment/test-flag.js
```
I'd expect that the folder structure would be
```
Webpack (webpack://)
* "webpack"
  * "bootstrap fc4578a0bef29d2a63f3"
* "src"
  * "components"
    * "index.js"
* ""
  * "Users"
    * "workspace"
      * "github"
        * "debugger.html"
          * "node_modules"
            * "react"
              * "index.js"
* "utils"
  * "debug.js"
* "node_modules"
  * "devtools-enviroment"
    * "test-flag.js"